article/sections/_4_graph_construction.tex

```diff
@@ -1,3 +1,90 @@
+We should add explicit nodes to our graph, subscripted or denoted by time, to show the cross-time causal relationships in our system.
+
+At this point, we have added all the outcome, explanatory, effect modifying, mediating, confounding, and time-indexed variables that we can think of to our causal graph.
+They are all disconnected nodes, singletons, in the graph.
+We now focus on pruning nodes from this graph, before drawing our final hypothesized connections.
+In particular, we focus on pruning ``post-outcome'' variables that are not part of the causal graph for future time periods or other observations.
+The reason for this is that conditioning on such post-outcome variables biases one's causal effect estimates.
+
+To remove the problematic variables, we iterate through each of the non-outcome variables in our graph, and we assess whether the variable is actually a result of the outcome (perhaps in combination with other variables in our graph).
+These post-outcome variables temporally follow the outcome variable(s) but do not cause variables in the causal graph for other observations.
+We remove all such post-outcome variables from our graph.
+
+Now is a good time to step back and consider what other researchers have thought.
+Specifically, we should conduct a literature review to see how other researchers have conceptualized the problem that we are working on.
+Have they included variables that we have not?
+Were those variables related our outcomes of interest?
+If so, should we add these variables to our causal graph, and how?
+Do the included variables of other researchers point suggest the existence of confounders in their work that we should include in our graph?
+Have other researchers ascribed differing roles to our graph's current variables than we have?
+For example, have other researchers judged a variable to be confounder, when we solely thought of the variable as an affect modifier?
+Critically examine the evidence for these alternative decisions to see if we should also reconsider how we're judging our variables.
+
+Finally, we need to connect the variables in our graph.
+\begin{enumerate}
+   \item Draw direct arrows from our explanatory variables, confounders, and effect modifiers to the outcomes.
+   \item Draw arrows from the explanatory variables to the mediators, and then draw arrows from the mediators to the outcomes.
+   \item Draw arrows from the confounders to the explanatory variables and mediators that they may cause.
+   \item Draw arrows from the variables in time $t$ to the variables that they cause in time $t+1$.
+\end{enumerate}
+After drawing in all arrows, we should now have a fully connected causal graph.
+Take a moment to look over the graph to ensure there are no remaining singletons and that we have not drawn any spurious connections.
+Also, take a moment to celebrate.
+Drawing one's first causal graph is hard work!
```

Comment:
The switch from "we" to talking directly to the reader is a bit weird. Should we stay consistent?

Reply:
Weird in the "oh snap, they're talking to ME!" sense or weird in the "whoa, this shouldn't happen sense?

I actually want the first kind of weird. I literally want the reader to pause and reflect a bit. I've edited this in the first revision to actually include a sentence that just says "[p]ause." Hopefully that will be enough of a visual cue that something different is going on, followed up by the linguistic cue in switching from point-of-view from us to the reader.

If you felt the second kind of weird, I'd definitely want to know why!

Reply:
It's the first type. Got it if that's the intent.